lib/pause_2017/PAUSE/Web/Plugin/RenderYAML.pm

```diff
@@ -12,6 +12,9 @@ sub register {
     local $YAML::Syck::ImplicitUnicode = 1;
     my $dump = YAML::Syck::Dump($data);
     my $edump = Encode::encode_utf8($dump);
+    my $action = $c->req->param('ACTION') || 'pause';
+    $c->res->headers->content_disposition("attachment; filename=$action.yaml");
```

Comment:
Mojo won't allow to set newlines in the header value, but I'd still be careful about directly injecting a request parameter into a response header.